build-llvm11.md

```diff
@@ -224,3 +212,24 @@ KLEE does not work under x86-32.
 **NOTE:** For testing real applications (e.g. Coreutils), you may need to increase your system's open file limit (ulimit -n).
 Something between 10000 and 999999 should work.
 In most cases, the hard limit will have to be increased first, so it is best to directly edit the corresponding configuration file (e.g., `/etc/security/limits.conf`).
+
+
+## Automated Installation (Ubuntu)
+
+This installation method is not officially supported and a byproduct of our continuous integration tests.
```

Comment:
Not true - of course it's officially supported. It's the other way around, the CI is just the byproduct.

It even has its full own description here in the official documentation: https://klee.github.io/build-script/ as part of the "Getting Started" section: https://klee.github.io/getting-started/.

Reply:
Should we even have this repetition?  We should just reference https://klee.github.io/build-script/ here, no?

Reply:
I added a link and would keep this section as https://klee.github.io/build-script/ is way too long to be helpful for a quick installation.

Reply:
Wait, how is it not helpful?

Reply:
It's too long. I'd nope out immediately if I want to install KLEE quickly. It's only helpful when you're modifying/benchmarking KLEE as a developer and need to understand the details.

Reply:
@251 It does have a TL;DR.  I would just have a link to that page, and improve the TL;DR section if you like. 
I think repeating information is bad.

Reply:
(and the TL;DR needs to be updated anyways to reference LLVM 9; again, the fewer places we need to update, the better)

Reply:
> It's too long. I'd nope out immediately if I want to install KLEE quickly.
If installing KLEE quickly is your goal, you shouldn't read this document in the first place.

Reply:
Ok, but https://klee.github.io/build-script/ is outdated in several places (Ubuntu 16.04, LLVM 9, ...).

Reply:
> If installing KLEE quickly is your goal, you shouldn't read this document in the first place.

That was my point. The scripts are extremely helpful on Ubuntu but they seem to be way too complicated by skimming that text.

Reply:
What's wrong with the TL;DR at that page?  My point is that you should modify that instead of duplicating it here. 
And yes, it is outdated, as I mentioned too, but it's better to have a single place to update rather than two.

Reply:
It's not duplicated anymore?!